CONTRIBUTING.MD

```diff
@@ -0,0 +1,116 @@
+Thanks for contributing!
+
+## Development
+
+### Installing dependencies
+
+```sh
+yarn install
+```
+
+You will find all building blocks that make up React Live in the [`src`](src) folder and examples in the [`demo`](demo) folder.
+
+### Testing
+
+You will find tests for files colocated with `*.test.js` suffixes. Whenever making any changes, ensure that all existing tests pass by running `yarn run test`.
+
+If you are adding a new feature or some extra functionality, you should also make sure to accompany those changes with appropriate tests.
+
+### Linting and Formatting
+
+Before committing any changes, be sure to do `yarn run lint`; this will lint all relevant files using [ESLint](http://eslint.org/) and report on any changes that you need to make.
+
+You will also want to ensure your code meets the prettier formatting guidelines by running `yarn run prettier -l <filename>` on a specific file. If there are differences the script errors out. You can also specify a glob `yarn run prettier -l "src/**/*.js"` which will return a list of files that do not conform.
+
+Alternatively, install the Prettier [editor plugin](https://prettier.io/docs/en/editors.html) in your favorite editor. This is the preferred method.
+
+There is also a CI step in place to lint all committed files. If any of the staged files do not conform to the eslint rules, your commit will fail until you resolve all outstanding issues. Prettier will be run automatically on your commit.
+
+### Before submitting a PR...
+
+Thanks for taking the time to help us make React-Live even better! Before you go ahead and submit a PR, make sure that you have done the following:
```

Comment:
```suggestion
Thanks for taking the time to help us make react-live even better! Before you go ahead and submit a PR, make sure that you have done the following:
```
super nit but react-live is lowercase everywhere else so it stuck out to me on this sentence!